Index: src/simulation/stage/CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/simulation/stage/CMakeLists.txt b/src/simulation/stage/CMakeLists.txt
--- a/src/simulation/stage/CMakeLists.txt	(revision 281ed731ed56b23d0879a2c765d4a20824a6d58a)
+++ b/src/simulation/stage/CMakeLists.txt	(date 1656128484137)
@@ -1,5 +1,5 @@
 cmake_minimum_required( VERSION 2.8.7 FATAL_ERROR )
-PROJECT(Stage)
+PROJECT(stage)

 SET( V_MAJOR 4 )
 SET( V_MINOR 3 )
